client/stylesheets/app.scss

```diff
@@ -1,20 +1,12 @@
-.app-content-fullscreen {
+.connection-status {
   position: fixed;
   top: 50px;
-  bottom: 0px;
   left: 0px;
   right: 0px;
+  z-index: 10000;
 }
 
-.connection-status-fullscreen {
-  position: fixed;
-  top: 50px;
-  left: 0px;
-  right: 0px;
-  z-index: 1;
-}
-
-.app-content-scrollable {
+body {
   padding-top: 70px;
```

Comment:
Did we arrive at this number via any particular set of principles?  Is this fundamentally 50px navbar + 20px content padding?

Reply:
This is taken straight from the bootstrap docs: https://getbootstrap.com/docs/3.4/components/#navbar-fixed-top

Reply:
(Added a comment)